src-v2/Judge/NTS.Judge.Blazor/Pages/Setup/Competitions/CompetitionForm.razor

```diff
@@ -35,7 +35,7 @@
     public DateTimeOffset CombineStartDayAndTime(DateTime? startDay, TimeSpan? startTime)
     {
         DateTime start = startDay.GetValueOrDefault(DateTime.MinValue);
-        DateTime startDayTime = start.Add((TimeSpan)startTime);
+        DateTime startDayTime = start.Date.Add((TimeSpan)startTime);
```

Comment:
GetValueOrDefault is a nice way to avoid an error when the nullable values are null but it will break the functionality of the application since the values are used within the logic. I think it is best to throw an error if the respective inputs contain null values.

Reply:
Unsure, which project do you reference by the name 'Not.Common'. Am I missing something?